member.php

```diff
@@ -443,16 +443,16 @@
 			switch($mybb->settings['username_method'])
 			{
 				case 0:
-					$emailmessage = $lang->sprintf($lang->email_randompassword, $user['username'], $mybb->settings['bbname'], $user_info['username'], $user_info['password']);
+					$emailmessage = $lang->sprintf($lang->email_randompassword, $user['username'], $mybb->settings['bbname'], $user_info['username'], $mybb->input['password']);
```

Comment:
Should probably use `$mybb->get_input()` here just for the sake of following what's used elsewhere.